app/controllers/events/view/speakers/create.js

```diff
@@ -26,7 +29,9 @@ export default class extends Controller {
       this.transitionToRoute('events.view.speakers', this.model.event.id);
     } catch (e) {
       console.error('Error while saving session', e);
-      this.notify.error(this.l10n.t('Oops something went wrong. Please try again'));
+      this.notify.error(this.l10n.t(this.errorHandler.genericError(e)));
```

Comment:
We won't return string from error handler, we'll just call the error handler on the error and it'll extract string and call notify.error

Reply:
used this.notify.error(this.l10n.t("error msg")) in error-handler.js and it's giving TypeError: Cannot read property 'error' of undefined

Reply:
Because you didn't inject the notify service